meteor-client/bundler.js

```diff
@@ -0,0 +1,78 @@
+var ChildProcess = require('child_process');
```

Comment:
Why should it be loaded as an external `npm` package? The `child_process.execFileSync` is available since `node@0.11.12`, which sound pretty stable to me. You probably say that because of the `cross-spawn`. Note that I don't use any `spawn` calls here, but rather `exec` calls, which doesn't have any problems across platforms.